### PR TITLE
Update airframes ref

### PR DESCRIPTION
**Pull request by hamishwillee**
Generated ref again from master source.

### CODE REVIEW

File: en/airframes/airframe_reference.md [hunk 1]
```diff
@@ -39,6 +39,34 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Dodecarotor cox
```
Comment (hamishwillee) [inline en/airframes/airframe_reference.md:42]: @bkueng There doesn't appear to be any image for this vehicle. Do you think we can get a build guide or other information?

**Comment by hamishwillee** [comment]
@bkueng Also, is it OK to build this off "master"? Ie obviously master leads the actual released code, so might differ from what people have on their vehicle.

File: en/airframes/airframe_reference.md (hunk 1 above)
Comment (bkueng) [reply en/airframes/airframe_reference.md:42]: It was recently added in https://github.com/PX4/Firmware/pull/7532.
@imcnanie if you could add a build log of the vehicle to the dev-guide here that would be great!

**Comment by bkueng** [comment]
> @bkueng Also, is it OK to build this off "master"? Ie obviously master leads the actual released code, so might differ from what people have on their vehicle.

Yes the information in the dev-guide is generally in sync with master. At some point we might want to introduce versioning.

File: en/airframes/airframe_reference.md (hunk 1 above)
Comment (hamishwillee) [reply en/airframes/airframe_reference.md:42]: @bkueng I mean a rotor-order diagram as per all the other copters in the airframe docs. As far as I can tell, not added in that PR. It is possible I am being dim!

Comment (bkueng) [reply en/airframes/airframe_reference.md:42]: You're right, that does not exist yet. I'll see if I can get it done, I'll need to check the motor assignments & directions.

**Pull request merged**